frontend/src/components/filter/activityReportFilters.js

```diff
@@ -380,7 +380,7 @@ export const specialistRoleFilter = {
 
 export const stateCodeFilter = {
   id: 'stateCode',
-  display: 'State',
+  display: 'State or Territory',
```

Comment:
![Screenshot 2023-08-03 at 9 04 03 AM](https://github.com/HHS/Head-Start-TTADP/assets/3288586/4c432489-9d67-444f-96f1-489c806219a6)

Should it be lowercase "t", as in```State or territory```? All the other filters are cased as such